### ISSUE TITLE
Limited to only four answers?

### ISSUE THREAD

**Issue by schonhose** [imladrisol/onlineExam#1]
Hi,

I discovered your repro today and since I am trying to achieve something similar I browsed your code. From what I have seen is that your database design limits the number of answers for the questions to 4.

I was wondering why the limit was imposed, imho you could have one column containing all the answers as a serialized array. That way there is virtually no limit to the number of answers. The only problem is that you need some more logic in the code to aggregate the results.

See for example [this package](https://github.com/fcosrno/exam-php)

Are you looking to emulate an exam or a webpolls also a possibility?

**Comment by imladrisol** [comment]
Yes, you're right. I will reorganize my tables structure as soon as possible. I also will thinking about webpolls. Thanks for suggestion.

**Comment by schonhose** [comment]
Webpolls are basically one-question exams without a predefined answer. I think you can save yourself some work when you can use the package I mentioned earlier.
